django_mongodb_backend/utils.py

```diff
@@ -28,7 +30,19 @@ def check_django_compatability():
         )
 
 
-def parse_uri(uri, *, db_name=None, test=None):
+def get_auto_encryption_options(crypt_shared_lib_path=None):
+    key_vault_database_name = "encryption"
+    key_vault_collection_name = "__keyVault"
+    key_vault_namespace = f"{key_vault_database_name}.{key_vault_collection_name}"
+    kms_providers = {"local": {"key": os.urandom(96)}}
+    return AutoEncryptionOpts(
+        key_vault_namespace=key_vault_namespace,
+        kms_providers=kms_providers,
+        crypt_shared_lib_path=crypt_shared_lib_path,
+    )
+
+
+def parse_uri(uri, *, auto_encryption_options=None, db_name=None, test=None):
```

Comment:
I'd rather than `options=None` than `auto_encryption_options=None` so that we're not blowing up the signature of `parse_uri()` with every `kwarg` of `MongoClient`. (If it's even in scope to continually expand `parse_uri()` rather than to guide more advanced users toward a dictionary `DATABASES`). I think all the changes in this file could be discussed/implemented in a follow up.

Reply:
Ah! Got it, thanks